### PR TITLE
Change rule that breaks responsive

### PR DESCRIPTION
**Pull request by aloupfor**
I removed a rule that breaks responsive on demo page.

Screeenshots:
Before
https://user-images.githubusercontent.com/11750180/37519655-4579d9a8-291a-11e8-9d14-9ccfe659a650.png

After
https://user-images.githubusercontent.com/11750180/37519660-48df8318-291a-11e8-85eb-cb8e9d7e4878.png

### CODE REVIEW

File: demo/styles/demo.scss [hunk 1]
```diff
@@ -72,7 +72,6 @@ section {
   }
 
   .tagline {
-    width: 450px;
```
Comment (kamranahmedse) [inline demo/styles/demo.scss]: I would rather add it back and set it to be `max-width: 450px;`

Comment (aloupfor) [reply demo/styles/demo.scss]: Done.
Can you tell me why do you rather have `max-width: 450px;`?

Comment (kamranahmedse) [reply demo/styles/demo.scss]: because that would fix the responsiveness and still keep the same behavior :)

Comment (aloupfor) [reply demo/styles/demo.scss]: Okay. Thank you for the answer.

**Pull request merged**